x-pack/platform/plugins/shared/fleet/public/applications/integrations/sections/epm/screens/detail/overview/markdown_renderers.tsx

```diff
@@ -90,6 +104,7 @@ export const markdownRenderers = (
         {children}
       </EuiLink>
     ),
+    // FIXME: https://github.com/remarkjs/react-markdown/blob/main/changelog.md#remove-extra-props-passed-to-certain-components
```

Comment:
`inline` is no longer passed as an extra prop for `code` components. https://github.com/remarkjs/react-markdown/blob/main/changelog.md#remove-extra-props-passed-to-certain-components